utils/test/check_submodule_commits

```diff
@@ -64,8 +74,10 @@ def main():
 def _parse_arguments():
     """Returns command line arguments"""
     parser = argparse.ArgumentParser(description=_DESCRIPTION)
-    msg = "Whether the commits should be updated in the CMakeLists.txt files"
-    parser.add_argument("-u", "--update", action='store_true', help=msg)
+    msgu = "Whether the commits should be updated in the CMakeLists.txt files"
+    parser.add_argument("-u", "--update", action='store_true', help=msgu)
+    msgc = "Check consistency between CMakeLists.txt and submodules"
```

Comment:
Any reason for not just reusing the `msg` variable? (Once `parser.add_argument()` had been called, the content of `msg` had been copied to an internal variable of the argument parser.)

Reply:
msg now used for both